docs/DeveloperGuide.md

```diff
@@ -86,7 +86,7 @@ We use [Cypress](https://www.cypress.io/) for automated end-to-end front-end tes
 ### Building and running RepoSense from code
 
 1. Execute the following command on the OS terminal inside the project directory. <br/>
-Usage: `gradlew run -Dargs="([--config CONFIG_FOLDER] | [--repos REPO_PATH_OR_URL...]) [--view [REPORT_FOLDER]] [--output OUTPUT_DIRECTORY] [--since DD/MM/YYYY] [--until DD/MM/YYYY] [--formats FORMAT...] [--ignore-standalone-config]"` <br/>
+Usage: `gradlew run -Dargs="([--config CONFIG_FOLDER] | [--repos REPO_PATH_OR_URL...]) [--view [REPORT_FOLDER]] [--output OUTPUT_DIRECTORY] [--since DD/MM/YYYY] [--until DD/MM/YYYY] [--formats FORMAT...] [--ignore-standalone-config] [--timezone UTC[±hh[mm]]]"` <br/>
```

Comment:
Here as well `--timezone ZONE_ID[±hh[mm]]`

Reply:
This part hasn't been updated?

Reply:
Reminder here, `--timezone ZONE_ID[±hh[mm]]`

Reply:
Sorry for not seeing this comment. Have corrected it.